.eslintrc.js

```diff
@@ -1,43 +1,34 @@
 module.exports = {
-	'env': {
-		'browser': true,
-		'commonjs': true,
-		'es2021': true,
-		'jest': true
-	},
-	'extends': 'eslint:recommended',
-	'overrides': [
-		{
-			'env': {
-				'node': true
-			},
-			'files': [
-				'.eslintrc.{js,cjs}'
-			],
-			'parserOptions': {
-				'sourceType': 'script'
-			}
-		}
-	],
-	'parserOptions': {
-		'ecmaVersion': 'latest'
-	},
-	'rules': {
-		'indent': [
-			'error',
-			'tab',
-		],
-		'linebreak-style': [
-			'error',
-			'unix'
-		],
-		'quotes': [
-			'error',
-			'single'
-		],
-		'semi': [
-			'error',
-			'always'
-		]
-	}
+  'env': {
+    'browser': true,
+    'commonjs': true,
+    'es2021': true,
+    'jest': true
+  },
+  'extends': 'eslint:recommended',
+  'overrides': [
+    {
+      'env': {
+        'node': true
+      },
+      'files': [
+        '.eslintrc.{js,cjs}'
+      ],
+      'parserOptions': {
+        'sourceType': 'script'
+      }
+    }
+  ],
+  'parserOptions': {
+    'ecmaVersion': 'latest'
+  },
+  'ignorePatterns': [
+    '**/*.test.js', '**/*.spec.js', '**/*.config.js', 'jest-setup.js', 'db.js'
+  ],
+  'rules': {
+    'indent': ['error', 2],
```

Comment:
This should be 'tab' - not 2

Reply:
tab does not work, we use this format for all repos and all workflows. It will not be changed.

Reply:
What do you mean 'tab does not work'? It has worked right up until now. And no we shouldn't be using 2 spaced in any other repos, if we do, that's a mistake

Reply:
@Jatewo please look at the workflow in educado-mobile